lcc_lokavaluto_app_connection/models/res_partner_backend.py

```diff
@@ -95,4 +97,16 @@ def get_wallet_balance(self):
             "success": False,
             "response": "No data - Please install financial backend Odoo add-on."
         }
-        return res
\ No newline at end of file
+        return res
+        
+    def get_wallet_commission_rule(self):
+        self.ensure_one()
+        rules = self.env["commission.rule"].search([("active", "=", True)])
+        for rule in rules:
+            # Get all the wallet matching the rule
+            wallets = self.search(safe_eval(rule.wallet_domain))
```

Comment:
I'm not sure to see how, without providing a context, this `safe_eval` is useful...

Reply:
well I don't know how it works exactly, but without the safe_eval() it doesn't work ^^'